### ISSUE TITLE
livecoin metadata is broken

### ISSUE THREAD

**Issue by npomfret** [timmolter/XChange#2222]
I've added a revert of #2217 to my PR because the addition of an invalid field breaks deserialisation.

**Comment by cymp** [comment]
wow that hurt, shouldn't we put an ignore tag on the fields in case it gets broken?

**Comment by npomfret** [comment]
That's one approach.  And it would allow addition of non-application fields (such as comments/notes).  

But it would facilitate people adding junk into these files as has been done.  The most minimal amount of testing would have shown this up.  I'd prefer to leave the system as is for now.

**Comment by cymp** [comment]
True, maybe we can add some tests calling the exchange creation though.
